news/2024/2024-05-29-osu-catch-world-cup-2024-round-of-32-recap.md

```diff
@@ -0,0 +1,94 @@
+
+The Round of 32 mappool is a clear step down in difficulty compared to the difficulty of the qualifier mappool. This makes it more comfortable to play for all teams and provides more exciting points between teams that are closely matched.
+
+The most picked map in the pool this weekend is Ironical Parade, the [HR1](https://osu.ppy.sh/beatmapsets/2183882#fruits/4615165) mapped by wwwww and Yoshi_green. The fairly conventional mapping with a focus on various stream patterns provided for a pleasant experience for both players and viewers. Our fruit catchers aren't very fond of the no-mod bracket as both [NM1](https://osu.ppy.sh/beatmapsets/2184377#fruits/4617105) and [NM4](https://osu.ppy.sh/beatmapsets/2184055#osu/4615558) only saw 3 appearances over all matches.
+
+## Round of 32 results
+
+In case you missed any of the matches, or are just curious about general results, you can view all of it here.
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| **Chile** ::{ flag=CL }:: | **5** | 0 | ::{ flag=NZ }:: New Zealand |
+| Sweden ::{ flag=SE }:: | 1 | **5** | **::{ flag=AU }:: Australia** |
+| **Russian Federation** ::{ flag=RU }:: | **5** | 0 | ::{ flag=VN }:: Vietnam |
+| **Indonesia** ::{ flag=ID }:: | **5** | 1 | ::{ flag=NO }:: Norway |
+| **China** ::{ flag=CN }:: | **5** | 3 | ::{ flag=NL }:: Netherlands |
+| **France** ::{ flag=FR }:: | **5** | 0 | ::{ flag=TR }:: Turkey |
+| **Canada** ::{ flag=CA }:: | **5** | 1 | ::{ flag=LV }:: Latvia |
+| **United Kingdom** ::{ flag=GB }:: | **5** | 0 | ::{ flag=MX }:: Mexico |
+| **Belgium** ::{ flag=BE }:: | **5** | 2 | ::{ flag=PH }:: Philippines |
+| **Taiwan** ::{ flag=TW }:: | **5** | 2 | ::{ flag=DE }:: Germany |
+| **Japan** ::{ flag=JP }:: | **5** | 0 | ::{ flag=TH }:: Thailand |
+| **Poland** ::{ flag=PL }:: | **5** | 0 | ::{ flag=MY }:: Malaysia |
+| **Finland** ::{ flag=FI }:: | **5** | 0 | ::{ flag=IS }:: Iceland |
+| **Argentina** ::{ flag=AR }:: | **5** | 2 | ::{ flag=DK }:: Denmark |
+| **United States** ::{ flag=US }:: | **5** | 0 | ::{ flag=EE }:: Estonia |
+| **Peru** ::{ flag=PE }:: | **5** | 0 | ::{ flag=ES }:: Spain |
+
+## Match highlights
+
+### ::{ flag=SE }:: Sweden vs. ::{ flag=AU }:: Australia
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2154793570&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Starting off the weekend early on Saturday, Sweden faced off against Australia. Sweden, seeded 14th, has joined the tournaments with high hopes as their roster this year managed to acquire their best qualification result since the format was introduced in 2022. However, their adversary Australia has managed to end up in the top 16 of the three past world cups and therefore isn't to be underestimated.
```

Comment:
```suggestion
Starting off the weekend early on Saturday, Sweden faced off against Australia. Sweden, seeded 14th, has joined the tournaments with high hopes as their roster this year managed to acquire their best qualification result since the format was introduced in 2022. However, their adversary Australia has managed to end up in the top 16 of the three past World Cups and therefore isn't to be underestimated.
```

Reply:
can be lowercase when it's in the general sense